index.html

```diff
@@ -1027,7 +1032,7 @@ <h4>serializeDerivedProofValue</h4>
 array of mandatory indexes (|mandatoryIndexes|), an array of
 selective indexes (|selectiveIndexes|), and a BBS presentation header
 (|presentationHeader|), the |featureOption| indicator, and possibly a
-|pseudonym| value depending on the |featureOption| value.
+|pseudonym| and |L| value depending on the |featureOption| value.
```

Comment:
Might `|pseudonym|` be calculated without `|L|`? Or vice versa? Or are they both always present when one is?
```suggestion
|pseudonym| and/or |L| value depending on the |featureOption| value.
```